eslint-rules/tests/component_deprecation.json

```diff
@@ -32,5 +32,11 @@
     "component": "List.Part",
     "source": "module-with-deprecations",
     "message": "Please use the 'List.Item' component instead."
+  },
+  {
+    "component": "TagsInput",
+    "source": "module-with-deprecations",
+    "message": "TagsInput was renamed, Please use 'ChipsInput' instead.",
+    "fix": {"componentName": "ChipsInput"}
   }
```

Comment:
These rules here are not going anywhere.. 
unlike our private lib when we use the rules for One App users. 
This file is just an example so there's no need to add a migration here.

Reply:
Yeah, I removed it :)